.github/workflows/tests.yml

```diff
@@ -38,7 +38,9 @@ jobs:
         run: composer config minimum-stability ${{ matrix.stability }}
 
       - name: Install symfony/flex
-        run: composer global require symfony/flex
+        run: |
+            composer global config --no-plugins allow-plugins.symfony/flex true
```

Comment:
I think we don't need to install Flex anymore with composer:v2

Reply:
TBH I've no idea [why it was installed in the first place](https://github.com/odolbeau/phone-number-bundle/pull/96).

Reply:
I think Flex was installed to have parallel download of packages